modules/nw-ingress-controller-configuration-parameters.adoc

```diff
@@ -41,6 +41,30 @@ If not set, the default value is based on `infrastructure.config.openshift.io/cl
 * GCP: `LoadBalancerService` (with External scope)
 * Bare metal: `NodePortService`
 * Other: `HostNetwork`
++
+[NOTE]
+====
+`HostNetwork` has a `hostNetwork` field with the following default values for the optional binding ports; `httpPort: 80`, `httpsPort: 443`, and `statsPort: 1936`.
```

Comment:
A lead-in to a list should use a colon rather than semicolon:
```suggestion
`HostNetwork` has a `hostNetwork` field with the following default values for the optional binding ports: `httpPort: 80`, `httpsPort: 443`, and `statsPort: 1936`.
```

Reply:
@bobfuru Shouldn't the enhancement request be brought to the Engineering team? >> please clarify I am not sure here as this EP is coming from engg.

Reply:
The change of adding the `NOTE` in this file is fine. It is not part of our auto-generated API reference docs. The other module is, however, and that's the one we are not able to hardcode directly.

Reply:
I ve created a new PR that touches only the first file and not the rest_api file